app/facades/menu/show_facade.rb

```diff
@@ -8,10 +8,18 @@ def new_order
       @order = @order.decorate
     end
 
+    def placed_order?(current_user)
+      menu.orders.find_by(user: current_user)
+    end
+
     def orders
       @orders ||= menu.orders.decorate
     end
 
+    def show_price?(current_user)
+      current_user.admin? && ordered_meals
```

Comment:
Use `ordered_meals.exists?`.
`where` will return an empty relation if there're no matches therefore `ordered_meals` will be always true